cmd/controller-manager/app/controllermanager.go

```diff
@@ -85,21 +87,18 @@ func Run(ctx context.Context, opts *options.Options) error {
 		LivenessEndpointName:       "/healthz",
 	})
 	if err != nil {
-		klog.Errorf("failed to build controller manager: %v", err)
-		return err
+		return fmt.Errorf("failed to build controller manager: %w", err)
```

Comment:
I can't see any benefits from the wrapped error here, but if the error happens it becomes a little complicated to figure out where the problem is.

I suggest letting this PR focus on what it meant to do.

Reply:
cc @lonelyCZ

Reply:
thanks, that also makes sense to me. Updated.